softwarereview_author.Rmd

```diff
@@ -11,29 +11,52 @@ This concise guide presents the software peer review process for you as a packag
+
+### Guidelines
+
+- Read and follow [our packaging style guide](#building), [reviewer's guide](#preparereview) to ensure your package meets our style and quality criteria.
+
+### Automatic checks
+
 - All submissions are automatically checked by our [pkgcheck](https://docs.ropensci.org/pkgcheck/) system to ensure packages follow our guidelines. All authors are expected to have run [the main `pkgcheck` function](https://docs.ropensci.org/pkgcheck/reference/pkgcheck.html) locally to confirm that the package is ready to be submitted. Alternatively, an even easier way to ensure a package is ready for submission is to use [the `pkgcheck` GitHub Action](https://github.com/ropensci-review-tools/pkgcheck-action) to run `pkgcheck` as a GitHub Action, as described in [our blog post](https://ropensci.org/blog/2022/02/01/pkgcheck-action/).
 -   If your package requires unusual system dependencies (see [*Packaging Guide*](#pkgdependencies)) for our GitHub Action to pass, please submit a pull request adding them to [our base Dockerfile](https://github.com/ropensci-review-tools/pkgcheck/blob/main/Dockerfile).
 - If there are any aspects of `pkgcheck` which your package is unable to pass, please explain reasons in your submission template.
-- If you feel your package is in scope for the
-  [Journal of Open-Source Software](https://joss.theoj.org/) (JOSS), do not submit it to JOSS consideration until after the rOpenSci review process is over: if your package is deemed in scope by JOSS editors, only the accompanying short paper would be reviewed, (not the software that will have been extended reviewed by rOpenSci by that time). Not all rOpenSci packages will meet the criteria for JOSS.
+
+### Accompanying manuscript (optional)
+
+If you intend to submit an accompanying manuscript for your package, rOpenSci has a collaborative partnership with the [Journal of Open-Source Software](https://joss.theoj.org/) and [Methods in Ecology and Evolution](https://besjournals.onlinelibrary.wiley.com/journal/2041210X):
+  
+- For a submission to Journal of Open-Source Software (JOSS), do not submit it to JOSS consideration until after the rOpenSci review process is over: if your package is deemed in scope by JOSS editors, only the accompanying short paper would be reviewed, (not the software that will have been extended reviewed by rOpenSci by that time). Not all rOpenSci packages will meet the criteria for JOSS.
+  
+- For a submission to Methods in Ecology and Evolution (MEE), submit it to MEE only after the rOpenSci review process is over, either before or after the package has been accepted. The review collaboration with MEE was introduced in a [blog post](https://ropensci.org/blog/2017/11/29/review-collaboration-mee/). The relevant article type for MEE is [Application](https://besjournals.onlinelibrary.wiley.com/hub/journal/2041210X/features/applicationpapers) for more details.
```

Comment:
I do not understand this text: first, we said, "Only send the package after our review is over," and then we said, "Either before or after the package has been accepted." So when should the author send it?  Does it send the package for review again, or does it have to send an article just like JOSS? It looks like It is the same as JOSS, so perhaps we could shorten this text and use it for both?  

If they are not the same, I don't understand the difference. 

I will wait for your changes to this text before continuing with the Spanish review.

Reply:
@robitalec did we mean accepted to CRAN?

Reply:
Aaah ok makes sense. How about for the sake of simplicity we do as if both preferred packages to be accepted? @mpadge